asm/main.asm

```diff
@@ -1869,145 +1873,159 @@ L_099C:
 	mov	a, #$ff
 	call	KeyOffVoices
 
-	inc	a
+	inc	a
 	call	SetEDLDSP		; Also set the delay to 0.
 	mov	$02, a			; 
 	mov	$06, a			; Reset the song number
-	mov	$0A, a			; 
+	mov	$0A, a			; 
 if !noSFX = !false
-	mov	$1d, a
+	mov	$1d, a
 endif
-	mov	!MaxEchoDelay, a	;
-	mov	a, #!reserveBufferZeroEDLGateDistance
+	mov	!MaxEchoDelay, a	;
+	mov	a, #!reserveBufferZeroEDLGateDistance
 	mov	SubC_table2_reserveBuffer_zeroEDLGate+1, a
 	call	EffectModifier
 
 	jmp	L_12F2             ; do standardish SPC transfer                                ;ERROR
 				; Note that after this, the program is "reset"; it jumps to wherever the 5A22 tells it to.
 				; The stack is also cleared.
-	;ret
+	;ret
 
 ProcessAPU1Input:				; Input from SMW $1DFA
-	mov	a, $01
-if !noSFX = !false
-if !useSFXSequenceFor1DFASFX = !false
-	beq	ProcessAPU1SFX
-else
-	bne	.hasCommand
-.terminate:
-	ret
-.hasCommand:
-endif
+	mov	a, $01
+if !noSFX = !false
+if !useSFXSequenceFor1DFASFX = !false
+	beq	ProcessAPU1SFX
+else
+	bne	.hasCommand
+.terminate:
+	ret
+.hasCommand:
+endif
 endif
 	cmp	a, #$ff
-	beq	L_099C
-if !noSFX = !true
-	cmp	a, #$08			; 08 unpauses music
-	beq	UnpauseMusic
-	cmp	a, #$07			; 07 pauses music
-	beq	PauseMusic
-	cmp 	a, #$09			; KevinM's edit:
+	beq	L_099C
+	cmp a, #$

```

Comment:
Did you forget something here? It looks like you forgot to delete something, or you meant to put a number here.